salt/cleanup/remove-post-orchestration.sls

```diff
@@ -0,0 +1,33 @@
+include:
+  - kubectl-config
+
+{%- set target = salt['pillar.get']('target') %}
+{%- set nodename = salt.caasp_net.get_nodename(host=target) %}
+
+###############
+# k8s cluster
+###############
+
+{%- set k8s_nodes = salt['mine.get']('roles:(kube-master|kube-minion|etcd)', 'nodename', 'grain_pcre').keys() %}
```

Comment:
Here we are usually explicitly saying `expr_form='grain_pcre'` (despite with a newer salt we'll have to change all occurences to `tgt_type`...)

Reply:
Yeah, I will change this just in case...

Reply:
It seems we are using both `expr_form` and `tgt_type`

```
...
salt/orch/kubernetes.sls:{%- set etcd_members = salt.saltutil.runner('mine.get', tgt='G@roles:etcd', fun='network.interfaces', tgt_type='compound').keys() %}
salt/orch/update.sls:{%- set all_masters = salt.saltutil.runner('mine.get', tgt=is_master_tgt, fun='network.interfaces', tgt_type='compound').keys() %}
salt/reboot/init.sls:{%- set etcd_members = salt['mine.get']('G@roles:etcd', 'nodename', expr_form='compound').values() %}
...
```

I will use `tgt_type` as it seems to be fully supported..